### PR TITLE
[SCSE-149] Import data from Wordpress

### PR DESCRIPTION
**Pull request by keykey117**
[SCSE-149] Added blogs.json and python script to import blogs onto payload.

### CODE REVIEW

File: apps/cms/src/import/import_script.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        }
+        ],
+        "status": "draft",
+        "createdAt": blogs[blog]["date"],
+        "updatedAt": blogs[blog]["date"]
+    }, indent=4)
+    headers = {
+        'Content-Type': 'application/json',
+        # need to reload cookie if it doesn't work
+        'Cookie': 'payload-token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImtleXlld0BnbWFpbC5jb20iLCJpZCI6IjYzZTNiMDVlOGVlNjU5MTgwMDE4MWIxOSIsImNvbGxlY3Rpb24iOiJ1c2VycyIsImlhdCI6MTY3NjIwNzc0MSwiZXhwIjoxNjc2MjE0OTQxfQ.t2zvMPnZIm7QqmTEefyIpX21FwxLI0oq0Ege5u6hPP4'
```
Comment (RealDyllon) [inline apps/cms/src/import/import_script.py:51]: Let's make this an env var

**Review by RealDyllon — CHANGES_REQUESTED** [review]

File: apps/cms/src/import/import_script.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import json
+import re
+
+# reading the blog posts from blogs.json
+with open('blogs.json') as json_file:
+    blogs = json.load(json_file)
+
+
+# importing the blogs onto payload
+url = "http://localhost:3000/api/posts"
```
Comment (RealDyllon) [inline apps/cms/src/import/import_script.py:12]: can make this an env var file.

File: apps/cms/src/import/import_script.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+print(len(blogs))
+for blog in blogs:
+
+    content = blogs[blog]["content"]
+    # remove html tags
+    content = re.sub("<(“[^”]*”|'[^’]*’|[^'”>])*>", "", content)
+    content = content.replace("\n", " \n")
+    payload = json.dumps({
+        "title": blogs[blog]["title"],
+        "author": "63e3b05e8ee6591800181b19",
```
Comment (RealDyllon) [inline apps/cms/src/import/import_script.py:22]: make this an env var too

**Review by RealDyllon — CHANGES_REQUESTED** [review]

**Comment by RealDyllon** [comment]
@keykey117 can rename this PR?

**Review by RealDyllon — DISMISSED** [review]
LGTM

**Review by jamiegoh — CHANGES_REQUESTED** [review]
can you add a readme about the required dependencies + .env vars needed like Blog_URL

**Review by jamiegoh — CHANGES_REQUESTED** [review]
can you combine the text in .env.example and requirements into the readme

**Review by jamiegoh — APPROVED** [review]
LGTM

**Pull request merged**